src/odemis/dataio/__init__.py

```diff
@@ -109,6 +110,9 @@ def find_fittest_converter(filename, default=tiff, mode=os.O_WRONLY, allowlossy=
     allowlossy: cf get_available_formats()
     returns (dataio. Module): the right exporter
     """
+    if any(filename.startswith(p) for p in catmaid.PREFIXES):
```

Comment:
It'd be quite nice if this module didn't need to import catmaid explicitly. I think that if you change this check into a loop similar to the way the extensions are handled, it'd feel a lot more generic, and also avoid depending on catmaid.
It'd look like:
```
fmt2ext = get_available_formats(mode, allowlossy)
for fmt in fmt2ext.keys():
   conv = get_converter(fmt)
   if hasattr(fmt, "PREFIXES") and any(filename.startswith(p) for p in conv.PREFIXES):
      return conv
```
Or maybe add an empty PREFIXES to all the other converter and so you can remove the `hasattr()`.